frontend/src/pages/CourseSelector/courseDescription/CourseDescription.jsx

```diff
@@ -26,6 +27,22 @@ const CourseAttribute = ({ title, content }) => {
   );
 };
 
+const ProgressBar = ({progress,height}) => {
+  let bgColor = "";
+  progress >= 75 ? bgColor = "red"
```

Comment:
These ternary statement is a bit confusing to read. Could you be able to simplify the logic? haha.

Something like this would work
```
const bgColor = "green"
// would be a cascading effect to assign the bgColor
progress >= 25 && (bgColor = "yellow")
progress >= 45 && (bgColor = "orange")
progress >= 75 && (bgColor = "red")
```

Also, if you have time, could you find similar hex codes to the yellow, orange and red color? I feel like the default css color options (i.e. red, yellow) have a higher constrast to what our main color theme is (generally a lighter and softer color feel)